Manifest.txt

```diff
@@ -1,8 +1,67 @@
+.rubocop.yml
+.rubocop_todo.yml
+.travis.yml
+Gemfile
 History.md
 LICENSE
 Manifest.txt
 README.md
 Rakefile
 bin/pmdtester
+config/all-java.xml
+config/design.xml
+config/project-list.xml
+config/project_list.txt
+config/projectlist_1_0_0.xsd
+lib/pmdtester/builders/diff_builder.rb
+lib/pmdtester/builders/diff_report_builder.rb
+lib/pmdtester/builders/html_report_builder.rb
+lib/pmdtester/builders/pmd_report_builder.rb
+lib/pmdtester/builders/rule_set_builder.rb
+lib/pmdtester/builders/summary_report_builder.rb
+lib/pmdtester/cmd.rb
 lib/pmdtester/parsers/options.rb
+lib/pmdtester/parsers/pmd_report_document.rb
+lib/pmdtester/parsers/projects_parser.rb
+lib/pmdtester/pmd_branch_detail.rb
+lib/pmdtester/pmd_error.rb
+lib/pmdtester/pmd_report_detail.rb
+lib/pmdtester/pmd_violation.rb
+lib/pmdtester/project.rb
+lib/pmdtester/report_diff.rb
+lib/pmdtester/resource_locator.rb
+lib/pmdtester/runner.rb
+resources/css/maven-base.css
+resources/css/maven-theme.css
+test/integration_test_pmd_report_builder.rb
```

Comment:
Should we really include the tests in the package? I think, they are not necessary....

Reply:
It seems, that rake doesn't include the test files.
Although on https://guides.rubygems.org/what-is-a-gem/ this is said:

> Inside gems are the following components:
> *    Code (including tests and supporting utilities)

I would actually remove the tests here. That way, if we add new tests, remove, rename, or so, we don't need to think about the package...